gcloud-java-dns/src/main/java/com/google/cloud/dns/spi/DefaultDnsRpc.java

```diff
@@ -36,6 +56,129 @@
   private final Dns dns;
   private final DnsOptions options;
 
+  private class DefaultRpcBatch implements RpcBatch {
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'batch' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2428393189/issues/source?bid=3123103&fileBranchId=3240468#l61)